.evg.yml

```diff
@@ -342,4 +375,5 @@ buildvariants:
   tasks:
     - name: run_core_tests
     - name: run_ios_tests
+    - name: run_ios_performance_tests
```

Comment:
[major q] Do we actually want to run these yet?

Reply:
I just wanted to test